### PR TITLE
[zh]Update content/zh/docs/concepts/workloads/controllers/cron-jobs.md

### PR DESCRIPTION
**Pull request by zyy19981018**

### CODE REVIEW

File: content/zh/docs/concepts/workloads/controllers/cron-jobs.md [hunk 1]
```diff
@@ -150,6 +150,39 @@ To generate CronJob schedule expressions, you can also use web tools like [cront
...
+Kubernetes that does not have experimental time zone support, all CronJobs in your cluster have an unspecified
+timezone).
+
+When you have the feature enabled, you can set `spec.timeZone` to the name of a valid [time zone](https://en.wikipedia.org/wiki/List_of_tz_database_time_zones) name. For example, setting
+`spec.timeZone: "Etc/UTC"` instructs Kubernetes to interpret the schedule relative to Coordinated Universal Time.
+
+A time zone database from the Go standard library is included in the binaries and used as a fallback in case an external database is not available on the system.
+-->
+
+＃＃ 时区
```
Comment (tengqm) [inline content/zh/docs/concepts/workloads/controllers/cron-jobs.md:170]:
```suggestion
## 时区 {#time-zones}
```

File: content/zh/docs/concepts/workloads/controllers/cron-jobs.md [hunk 2]
```diff
@@ -150,6 +150,39 @@ To generate CronJob schedule expressions, you can also use web tools like [cront
...
+对于没有指定时区的 CronJobs，kube-controller-manager 解释相对于其本地时区的计划。
```
Comment (tengqm) [inline content/zh/docs/concepts/workloads/controllers/cron-jobs.md:171]:
```suggestion
对于没有指定时区的 CronJob，kube-controller-manager 基于本地时区解释排期表。
```

File: content/zh/docs/concepts/workloads/controllers/cron-jobs.md [hunk 3]
```diff
@@ -150,6 +150,39 @@ To generate CronJob schedule expressions, you can also use web tools like [cront
...
+
+{{< feature-state for_k8s_version="v1.24" state="alpha" >}}
+
+如果启用 `CronJobTimeZone` [特性门控](/docs/reference/command-line-tools-reference/feature-gates/)，
```
Comment (tengqm) [inline content/zh/docs/concepts/workloads/controllers/cron-jobs.md:175]:
```suggestion
如果启用 `CronJobTimeZone` [特性门控](/zh/docs/reference/command-line-tools-reference/feature-gates/)，
```

File: content/zh/docs/concepts/workloads/controllers/cron-jobs.md [hunk 4]
```diff
@@ -150,6 +150,39 @@ To generate CronJob schedule expressions, you can also use web tools like [cront
...
+你可以为 CronJob 指定一个时区（如果您没有启用该特性门控，或者你使用的是
+不支持实验时区的 Kubernetes，集群中的所有 CronJobs 都有一个未指定的
+时区）。
```
Comment (tengqm) [inline content/zh/docs/concepts/workloads/controllers/cron-jobs.md:178]: 请在中英文边界或标点符号处换行。

```suggestion
你可以为 CronJob 指定一个时区（如果你没有启用该特性门控，或者你使用的是不支持试验性时区功能的
Kubernetes 版本，集群中所有 CronJob 的时区都是未指定的）。
```

File: content/zh/docs/concepts/workloads/controllers/cron-jobs.md [hunk 5]
```diff
@@ -150,6 +150,39 @@ To generate CronJob schedule expressions, you can also use web tools like [cront
...
+
+启用该功能后，你可以将 `spec.timeZone` 设置为有效 [时区](https://en.wikipedia.org/wiki/List_of_tz_database_time_zones) 名称的名称。 例如，设置
```
Comment (tengqm) [inline content/zh/docs/concepts/workloads/controllers/cron-jobs.md:180]:
```suggestion
启用该特性后，你可以将 `spec.timeZone`
设置为有效[时区](https://en.wikipedia.org/wiki/List_of_tz_database_time_zones)名称。 
例如，设置
```

File: content/zh/docs/concepts/workloads/controllers/cron-jobs.md [hunk 6]
```diff
@@ -150,6 +150,39 @@ To generate CronJob schedule expressions, you can also use web tools like [cront
...
+`spec.timeZone: "Etc/UTC"` 指示 Kubernetes 解释相对于协调世界时的时间表。
```
Comment (tengqm) [inline content/zh/docs/concepts/workloads/controllers/cron-jobs.md:181]:
```suggestion
`spec.timeZone: "Etc/UTC"` 指示 Kubernetes采用 UTC 来解释排期表。
```

File: content/zh/docs/concepts/workloads/controllers/cron-jobs.md [hunk 7]
```diff
@@ -150,6 +150,39 @@ To generate CronJob schedule expressions, you can also use web tools like [cront
...
+
+Go 标准库中的时区数据库包含在二进制文件中，并用作备用数据库，以防系统上没有可用的外部数据库
```
Comment (tengqm) [inline content/zh/docs/concepts/workloads/controllers/cron-jobs.md:183]:
```suggestion
Go 标准库中的时区数据库包含在二进制文件中，并用作备用数据库，以防系统上没有可用的外部数据库。
```

Comment (zyy19981018) [reply content/zh/docs/concepts/workloads/controllers/cron-jobs.md]: thanks

**Comment by tengqm** [comment]
Several things:

- Please help mark each feedback as "resolved" after fixes are applied;
- Please always keep your PR as a single commit, we can help review that, but mistakes are still possible

/approve

File: content/zh/docs/concepts/workloads/controllers/cron-jobs.md [hunk 8]
```diff
@@ -150,6 +150,39 @@ To generate CronJob schedule expressions, you can also use web tools like [cront
...
+对于没有指定时区的 CronJob，kube-controller-manager 基于本地时区解释排期表。
+
+{{< feature-state for_k8s_version="v1.24" state="alpha" >}}
+
+如果启用 `CronJobTimeZone` [特性门控](/zh/docs/reference/command-line-tools-reference/feature-gates/)，
+你可以为 CronJob 指定一个时区（如果你没有启用该特性门控，或者你使用的是不支持试验性时区功能的
+Kubernetes 版本，集群中所有 CronJob 的时区都是未指定的）。
+
+启用该特性后，你可以将 `spec.timeZone`
+设置为有效[时区](https://zh.wikipedia.org/wiki/List_of_tz_database_time_zones)名称。
```
Comment (Sea-n) [inline content/zh/docs/concepts/workloads/controllers/cron-jobs.md:180]: This wiki article have no Mandarin version. Even there have one, it should have different URL path.

File: content/zh/docs/concepts/workloads/controllers/cron-jobs.md [hunk 9]
```diff
@@ -150,6 +150,39 @@ To generate CronJob schedule expressions, you can also use web tools like [cront
...
+
+{{< feature-state for_k8s_version="v1.24" state="alpha" >}}
+
+如果启用 `CronJobTimeZone` [特性门控](/zh/docs/reference/command-line-tools-reference/feature-gates/)，
+你可以为 CronJob 指定一个时区（如果你没有启用该特性门控，或者你使用的是不支持试验性时区功能的
+Kubernetes 版本，集群中所有 CronJob 的时区都是未指定的）。
+
+启用该特性后，你可以将 `spec.timeZone`
+设置为有效[时区](https://zh.wikipedia.org/wiki/List_of_tz_database_time_zones)名称。 
+例如，设置 `spec.timeZone: "Etc/UTC"` 指示 Kubernetes采用 UTC 来解释排期表。
```
Comment (Sea-n) [inline content/zh/docs/concepts/workloads/controllers/cron-jobs.md:181]: nit: There should be one space between `Kubernetes 采用`.

**Review by Sea-n — CHANGES_REQUESTED** [review]

File: content/zh/docs/concepts/workloads/controllers/cron-jobs.md (hunk 8 above)
Comment (ydFu) [inline content/zh/docs/concepts/workloads/controllers/cron-jobs.md:180]:
```suggestion
设置为有效[时区](https://zh.wikipedia.org/zh-hant/%E6%97%B6%E5%8C%BA%E4%BF%A1%E6%81%AF%E6%95%B0%E6%8D%AE%E5%BA%93s)名称。 
```

File: content/zh/docs/concepts/workloads/controllers/cron-jobs.md (hunk 9 above)
Comment (ydFu) [inline content/zh/docs/concepts/workloads/controllers/cron-jobs.md:181]:
```suggestion
例如，设置 `spec.timeZone: "Etc/UTC"` 指示 Kubernetes 采用 UTC 来解释排期表。
```

File: content/zh/docs/concepts/workloads/controllers/cron-jobs.md (hunk 8 above)
Comment (Sea-n) [reply content/zh/docs/concepts/workloads/controllers/cron-jobs.md:180]: Maybe this URL is better. (Change `zh-hant` to user's default locale)
https://zh.wikipedia.org/wiki/%E6%97%B6%E5%8C%BA%E4%BF%A1%E6%81%AF%E6%95%B0%E6%8D%AE%E5%BA%93

**Comment by tengqm** [comment]
@zyy19981018 Would you please move this forward by addressing the feedback?

File: content/zh/docs/concepts/workloads/controllers/cron-jobs.md [hunk 10]
```diff
@@ -150,6 +151,39 @@ To generate CronJob schedule expressions, you can also use web tools like [cront
...
+timezone).
+
+When you have the feature enabled, you can set `spec.timeZone` to the name of a valid [time zone](https://en.wikipedia.org/wiki/List_of_tz_database_time_zones) name. For example, setting
+`spec.timeZone: "Etc/UTC"` instructs Kubernetes to interpret the schedule relative to Coordinated Universal Time.
+
+A time zone database from the Go standard library is included in the binaries and used as a fallback in case an external database is not available on the system.
+-->
+
+## 时区 {#time-zones}
+对于没有指定时区的 CronJob，kube-controller-manager 基于本地时区解释排期表。
```
Comment (tengqm) [inline content/zh/docs/concepts/workloads/controllers/cron-jobs.md:172]:
```suggestion
对于没有指定时区的 CronJob，kube-controller-manager 基于本地时区解释排期表（Schedule）。
```

File: content/zh/docs/concepts/workloads/controllers/cron-jobs.md [hunk 11]
```diff
@@ -150,6 +151,39 @@ To generate CronJob schedule expressions, you can also use web tools like [cront
...
+
+{{< feature-state for_k8s_version="v1.24" state="alpha" >}}
+
+如果启用 `CronJobTimeZone` [特性门控](/zh/docs/reference/command-line-tools-reference/feature-gates/)，
```
Comment (tengqm) [inline content/zh/docs/concepts/workloads/controllers/cron-jobs.md:176]:
```suggestion
如果启用了 `CronJobTimeZone` [特性门控](/zh/docs/reference/command-line-tools-reference/feature-gates/)，
```

**Pull request merged**